profile_bluesky/startup/29-axis_tuning.py

```diff
@@ -549,3 +567,47 @@ def compute_tune_ranges():
         ms_stage.rp.tuner.width = 3
         as_stage.rp.tuner.width = 3
         yield from bps.mv(terms.USAXS.usaxs_minstep, 0.000025)
+
+
+def update_EPICS_tuning_widths():
+    """
+    plan: update the tuning widths in EPICS PVs from local settings
+    """
+    yield from bps.mv(
+        axis_tune_range.mr,     m_stage.r.tuner.width,
+        axis_tune_range.ar,     a_stage.r.tuner.width,
+        axis_tune_range.m2rp,   m_stage.r2p.tuner.width,
+        axis_tune_range.a2rp,   a_stage.r2p.tuner.width,
+        axis_tune_range.msrp,   ms_stage.rp.tuner.width,
+        axis_tune_range.asrp,   as_stage.rp.tuner.width,
+        axis_tune_range.dx,     d_stage.x.tuner.width,
+        axis_tune_range.dy,     d_stage.y.tuner.width,
+        )
+
+
+def user_defined_settings():
+    """
+    plan: users may redefine this function to override any instrument defaults
```

Comment:
Can this be clearer?  Needs to be all on one line, though.

Users who redefine this should put that new definition in a python file in the *user area*.